package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rvw_developers_core",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Can you also update the package-lock.json file ?

Reply:
what in the lock file needs updated?

Reply:
done.